docs/extend/modular-resources/examples/csi.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Add a CSI Camera as a Modular Resource"
+linkTitle: "CSI Camera"
+weight: 40
+type: "docs"
+description: "Use the viam:camera:csi model to add a CSI Camera to your robot."
+tags: ["board", "csi", "jetson", "serial", "module", "modular resources", "Python", "python SDK", "nvidia", "jetson orin", "jetson orin nano", "nano", "camera"]
+# SMEs: Sean
+---
+
+
+Many boards, like the Jetson Orin Nano, come with built-in Camera Serial Interface, or CSI, cameras.
+These cameras are excellent for utilizing embedded vision systems like Viam's [vision service](/services/vision/).
+If after configuring a `jetson` board you want to add your Orin Nano's CSI camera to your robot, you can.
+Viam provides a modular resource [extending](/extend/modular-resources/) the [camera API](/components/camera/#api) as a new `viam:camera:csi` model of [camera](/components/camera/).
```

Comment:
```suggestion
Not all CSI cameras are supported by the [webcam camera model](/components/camera/webcam).
Instead, Viam supports CSI cameras by providing a {{< glossary_tooltip term_id="modular resource" text="modular resource" >}} for your CSI camera: `viam:camera:csi`.
```
You say this but in a slightly roundabout way - that Viam provides a modular resource [extending](/extend/modular-resources/) the [camera API](/components/camera/#api) as a new `viam:camera:csi` model of [camera](/components/camera/).
```

Reply:
LGTM, done